package.json

```diff
@@ -10,16 +10,22 @@
     "test": "jest --watchAll"
   },
   "dependencies": {
+    "@nextui-org/accordion": "^2.0.38",
+    "@nextui-org/system": "^2.2.5",
+    "@nextui-org/theme": "^2.2.9",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "framer-motion": "^11.3.22",
     "install": "^0.13.0",
```

Comment:
it looks to me like you installed this dependency by accident, to get rid of it, you can type

`npm uninstall install` 

to your terminal